regulations/static/regulations/css/less/module/drawer.less

```diff
@@ -187,7 +187,7 @@ currently these are TOC, History, and Search
     .close & {
         display: block;
         border-right: none;
-        width: 40px;
+        width: 39px;
```

Comment:
Magic numbers frighten me. Can we track down where the `40px` came from, put that in a variable, and then use `$that_var - 1` here? That'd explain the _why_ much better, if that makes sense.